topostats/validation.py

```diff
@@ -262,6 +262,7 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
                 error="Invalid value in config for 'grainstats.extract_height_profile',"
                 "valid values are 'True' or 'False'",
             ),
+            "class_names": [str],
```

Comment:
Could we be more explicit here and state what we expect these to be with...

```suggestion
            "class_names": Or(
                         "DNA",
                         "Protein",
                         error="Invalid value in config for 'class_names'"
                 ),
```

...although that probably isn't the way to validate a list but I've more to suggest on this which I'll write in the broader comment.